docs/connected-accounts.rst

```diff
@@ -66,6 +65,38 @@ Repository status (``repo:status``)
     but there isn't a more granular permission
     that only allows setting up SSH keys for read access.
 
+.. _github-permission-troubleshooting:
```

Comment:
We are using autolabel, there is no need for this, at least I'm missing something?

Reply:
I think I added this prior to adding the section label.